2-mandatory/2-financial-times.js

```diff
@@ -5,7 +5,12 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+  const shortArticleTitles = [];
+    for (let articleTitle of allArticleTitles) {
```

Comment:
indentation seem off here too.

Reply:
Good job overall code inside the function will work correct but it doesn't seem to output a value. You are definitely missing something here.